src/tetris/scene/playScene.ts

```diff
@@ -219,10 +235,13 @@ export default class PlayScene extends Phaser.Scene {
 	}
 
 	private _exitMatch(): void {
-		this._game.handleEndOfMatch(this._match);
+		this._game.changeScene(config.sceneKeys.menuScene);
```

Comment:
Usually, changing scene took place in `handleEndOfMatch()`.

Reply:
In current version of the developer, there's no such call. The only subscriber to the endOfMatch event is the profiler, which definitely doesnt change the scene.

Reply:
Ahh sorry, mixed it up with the ExitButton (exitGame)

Reply:
👌